samples/matter/weather_station/weather-station.zap

```diff
@@ -0,0 +1,6998 @@
+{
```

Comment:
What is this `zap` file, other samples don't seem to include it?

Reply:
It's a file that is used by the ZAP tool to generate files located in the `gen` directory. Other samples doesn't have it, as they have their corresponding light and lock samples in the upstream, so .zap files are located in there and that one sample is unique for the NCS. Nevertheless that was mistake from our side that we didn't include those .zap files for all samples from beginning and we are going to fix that soon, so all NCS samples will contain such files to allow easily regenerating `gen` directory content.

Reply:
Ok, thanks for exaplaination. Does this format support some kind of comments? Would it be possible to add license header to it?

Reply:
I'm afraid not, this seems to be format inherited from .json that also doesn't support comments I guess.

Reply:
ok, thanks for confirming 👍